frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -403,7 +413,7 @@ export function CredentialsForm(
         stringData.additionalTrustBundle = additionalTrustBundle
         break
       case Provider.openstack:
-        stringData['clouds.yaml'] = enforceCloudsYaml(cloudsYaml, cloud, osCABundle)
+        stringData['clouds.yaml'] = cloudsYaml
```

Comment:
Is there a bug you are solving by moving this logic, rather than just fixing the YAML structure? I remember this had very specific behaviour, so I worry about changing it.

Reply:
yes -- by doing it on line 406 it updates just the yaml file and not the wizard ui input for "clouds.yaml"